islandora-connector-broadcast/src/main/cfg/ca.islandora.alpaca.connector.broadcast.cfg

```diff
@@ -0,0 +1,7 @@
+# Incoming queue/topic on the broker of your choice.
+# Replace the 'broker' prefix with the broker for your system.
+# E.g. activemq, zeromq, etc... 
+input.stream=broker:queue:islandora-connector-broadcast
+
+# Header that contains the recipient list
+recipients.header=ca.islandora.alpaca.connector.broadcast.recipients
```

Comment:
And you may decide on something like `CamelAlpacaBroadcasterRecipients` or the like -- whatever you think best fits.